README.md

```diff
@@ -145,6 +187,47 @@ The Unix timestamp in seconds at which the access and client tokens will expire.
 
 Creates a tokens object from a refresh token, which can be passed to higher-level SDK methods. The SDK will handle refreshing the tokens for accessing relevant data.
 
+### `ImageUploadOpts`
+
+#### Properties
+
+##### `images: EncodedImage[]`
+
+Required. An array of `EncodedImage` objects representing images to upload.
+
+### `EncodedImage`
```

Comment:
Should we add support for the `SupplementaryData` and `ExtractedBarcode`

Reply:
Going to kick this to another PR